packages/cli/src/util/validate-paths.ts

```diff
@@ -1,14 +1,11 @@
-import { lstat as lstatRaw } from 'fs';
-import { promisify } from 'util';
+import { lstat } from 'fs-extra';
```

Comment:
**convention (non-blocking):** ‏What do you think about using the built-in Node.js version?

```suggestion
import { lstat } from 'fs/promises';
```

Reply:
I don't think it makes much difference since we're already using `fs-extra` throughout the code base.